diff --git a/node_modules/nativewind/src/tailwind/native.ts b/node_modules/nativewind/src/tailwind/native.ts
index b7e9b1c..96c0ad6 100644
--- a/node_modules/nativewind/src/tailwind/native.ts
+++ b/node_modules/nativewind/src/tailwind/native.ts
@@ -171,6 +171,17 @@ const nativePlugins = plugin(function ({
     },
   });
 
+  addUtilities({
+    ".border-continuous": {
+      "@rn-move -rn-border-curve border-curve": "true",
+      "-rn-border-curve": "continuous",
+    },
+    ".border-circular": {
+      "@rn-move -rn-border-curve border-curve": "true",
+      "-rn-border-curve": "circular",
+    }
+  });
+
   /**
    * https://reactnative.dev/docs/pressable#rippleconfig
    */
@@ -319,6 +330,10 @@ const preset: Config = {
       borderWidth: {
         hairline: hairlineWidth(),
       },
+      borderCurve: {
+        continuous: "continuous",
+        circular: "circular",
+      },
       letterSpacing: {
         tighter: "-0.5px",
         tight: "-0.25px",
