diff --git a/node_modules/react-native-reanimated/src/hook/useAnimatedStyle.ts b/node_modules/react-native-reanimated/src/hook/useAnimatedStyle.ts
index 877d8d9..d1a7eee 100644
--- a/node_modules/react-native-reanimated/src/hook/useAnimatedStyle.ts
+++ b/node_modules/react-native-reanimated/src/hook/useAnimatedStyle.ts
@@ -570,7 +570,7 @@ For more, see the docs: \`https://docs.swmansion.com/react-native-reanimated/doc
         );
       };
     }
-    if (animatedUpdaterData.current) {
+    if (animatedUpdaterData.current?.styleUpdaterContainer?.current) {
       animatedUpdaterData.current.styleUpdaterContainer.current = fun;
     }
     const mapperId = startMapper(fun, inputs);
